src/System.Net.Primitives/tests/FunctionalTests/System.Net.Primitives.Functional.Tests.csproj

```diff
@@ -33,13 +33,14 @@
     <Compile Include="IPAddressMappingTest.cs" />
     <Compile Include="IPEndPointTest.cs" />
     <Compile Include="NetworkCredentialTest.cs" />
-    <Compile Include="RequestCachePolicyTest.cs" Condition="'$(TargetGroup)' == ''" />
     <Compile Include="SocketAddressTest.cs" />
   </ItemGroup>
   <ItemGroup Condition="'$(TargetGroup)' == ''">
     <Compile Include="LoggingTest.cs" />
     <Compile Include="SerializationTest.cs" />
+    <Compile Include="RequestCachePolicyTest.cs" />
     <Compile Include="CookieContainerAddTest.cs" />
+    <Compile Include="NetworkCredentialTest.netstandard1.7.cs" />
```

Comment:
Yes netstandard2.0 is - netstandard1.7 but in all of our projects we have being calling our files that are netstandard1.7 specific with that name alias.  

And yes I think that AOT/UWP is stil based on netstandard1.3 so thats why we have specific tests.

Thank you for your review @CIPop